modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -703,6 +703,10 @@ private function is_active_icon_exist( $settings ):bool {
 	}
 
 	private function render_accordion_icons( $settings ) {
+		if ( '' === $settings['accordion_item_title_icon']['value'] ) {
```

Comment:
```suggestion
		if ( empty( $settings['accordion_item_title_icon']['value'] ) ) {
```

Reply:
@Barcov You can maybe even put this `if()` in line 761. Then this function does not need an early return. And also if `$settings['accordion_item_title_icon']['value']` is not empty, then there will always be content returned, which is why you can replace the if() on line 761 with this if(). WDYT?

Reply:
@louiswol94 Yes I like that suggestion. I generally try to keep the html portion of render quite lean. But in this case the if on 706 does feel redundant.